test/images/agnhost/guestbook/guestbook.go

```diff
@@ -50,7 +49,7 @@ var (
 	httpPort    string
 	backendPort string
 	slaveOf     string
-	slaves      []string
+	slaves      []net.Conn
```

Comment:
Not related to this PR but we have built our own app here right? We could name these something else...

Reply:
You mean rename this var?

Reply:
yeah, I filed a change for this thinking it should be straightforward ... looks like we still need to get the image pushing fixed though 😬

Reply:
I am holding that change for now to avoid creating unnecessary conflict with your work and to let the image build be sorted out first. I intend to merge it at some point https://github.com/kubernetes/kubernetes/pull/87105